.github/workflows/job_deploy_api_enterprise.yaml

```
@@ -9,12 +9,16 @@ on:
 jobs:
   deploy:
-    runs-on: blacksmith-4vcpu-ubuntu-2404
+    runs-on: ubuntu-latest
     environment: Enterprise
     strategy:
       fail-fast: true
       matrix:
-        workers: [{name: "enterprise-default", domain: "enterprise.unkey.dev"}, {name: "enterprise-fireworks", domain: "fireworks.unkey.dev"}]
+        workers:
+          [
+            { name: "enterprise-default", domain: "enterprise.unkey.dev" },
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)